cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -55,6 +55,7 @@ public ClusterOperator(String namespace,
                            KafkaConnectAssemblyOperator kafkaConnectAssemblyOperator,
                            KafkaConnectS2IAssemblyOperator kafkaConnectS2IAssemblyOperator,
                            KafkaMirrorMakerAssemblyOperator kafkaMirrorMakerAssemblyOperator) {
+        log.info("You are proud user of Strimzi version {}", getClass().getPackage().getImplementationVersion());
```

Comment:
I don't think this is the right place. You should print it somewhere in `Main.java`. Here it will be printed for every single namespace the CO serves. In `Main` you can print it just once. For trouble shooting purposes you also need this to be printed regardless of what happens later. So directly in the `main` method sounds reasonable - you can just add it to the message _ClusterOperator is starting_ which we already have there ... _ClusterOperator 0.9.0-SNAPSHOT is starting_.